api/python/quilt3/packages.py

```diff
@@ -311,6 +311,8 @@ def physical_keys(self):
 class Package(object):
     """ In-memory representation of a package """
 
+    use_tqdm = os.getenv('QUILT_USE_TQDM').lower() == 'true'
```

Comment:
Why not just import this from `data_transfer`? As a rule copying the same code twice is not a good idea. Also: please run `pylint` on all files in this PR.